Unit/Suite.php

```diff
@@ -59,6 +59,14 @@ public function __construct()
         $protocol['Test']        = new Core\Protocol\Generic('Test', null);
         $protocol['Test']['Vfs'] = new Test\Protocol\Vfs();
 
+        $constantMocker = new Test\Mocker\Constant($this->getPhpMocker());
+        $this->getAssertionManager()->setPropertyHandler(
+            'constant',
+            function () use ($constantMocker) {
+                return $constantMocker;
```

Comment:
You mean, having the initialisation inside the function instead of outside? Why?